queue-version.lua

```diff
@@ -0,0 +1,2 @@
+-- Сontains the module version, requires manual update.
```

Comment:
So what's our proposed plan here? Bump the version once per tag? Or something a bit more complicated like https://github.com/tarantool/metrics/pull/426/commits/b63093938a2988611bd568220fae6281f106c493 ?

Reply:
Let's use https://github.com/tarantool/metrics/commit/b63093938a2988611bd568220fae6281f106c493  approach

Reply:
We've discussed the question with @LeonidVas and decided to stick to the simplest approach -- just bump the version for a tag commit, do not mark scm version in any way.

Reply:
Done